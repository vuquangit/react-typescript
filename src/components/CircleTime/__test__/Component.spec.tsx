/**
 *
 * Tests for MyComponent
 *
 */

import React from 'react'
import { render /* fireEvent */ } from '@testing-library/react'

import Component from '..'
import AppWrapper from '@/test/supports/AppWrapper'

describe('App SwitchTheme', () => {
  it('Test SwitchTheme with default', async () => {
    const { container, getByTestId } = render(
      <AppWrapper themeMode="light">
        <Component />
      </AppWrapper>
    )

    expect(container).toMatchSnapshot()
  })
})
