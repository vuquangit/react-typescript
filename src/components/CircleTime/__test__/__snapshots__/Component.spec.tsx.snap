// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App SwitchTheme Test SwitchTheme with default 1`] = `
.c0 {
  position: relative;
  height: 100%;
  overflow: auto;
  -webkit-overflow-scrolling: touch;
}

.c1 {
  position: relative;
  text-align: center;
  min-height: 300px;
  max-height: 700px;
  height: 80%;
  border: 1px solid #000;
}

.c2 {
  padding: 10px;
}

<div>
  <div
    class="c0 Loop js-loop"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        01
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        02
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        03
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        04
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        05
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        06
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        07
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        08
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        09
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        10
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        11
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        12
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        13
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        14
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        15
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        16
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        17
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        18
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        19
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        20
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        21
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        22
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        23
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        24
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        25
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        26
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        27
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        28
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        29
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        30
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        31
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        32
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        33
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        34
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        35
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        36
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        37
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        38
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        39
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        40
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        41
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        42
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        43
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        44
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        45
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        46
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        47
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        48
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        49
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        50
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        51
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        52
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        53
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        54
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        55
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        56
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        57
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        58
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        59
      </div>
    </div>
    <div
      class="c1 green is-clone"
    >
      <div
        class="c2"
      >
        01
      </div>
    </div>
    <div
      class="c1 red is-clone"
    >
      <div
        class="c2"
      >
        02
      </div>
    </div>
  </div>
</div>
`;
